docs/index.rst

```diff
@@ -9,9 +9,10 @@ resources targeted for edX learners, educators, researchers, and Open edX operat
     :hidden:
 
     Open edX Proposals <https://open-edx-proposals.readthedocs.io/en/latest/>
-    Open edX Developer's Guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/>
+    developers-guide/*
```

Comment:
This needs to be a link, doesn't it?

Reply:
I have a typo, but it looks like it scans by slug: https://shimizukawa-sphinx.readthedocs.io/en/latest/markup/toctree.html